cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -121,23 +130,60 @@ function Autocomplete( element, opts ) {
       inputCoords.left ) + 'px';
   }
 
+  /**
+   * Toggles the error state and message that's shown when the
+   * max search term length is hit
+   */
+  function _toggleMaxLengthError() {
+    _maxLengthExceeded = _searchTerm.length >= _settings.maxChars ? true :
+      false;
+    if ( _maxLengthExceeded ) {
+      _input.classList.add( ERROR_CLASS );
+
+      /* If you type fast enough, search results from a letter or two back can
+         cover up the max length error, so we'll wait a bit before hiding */
+      setTimeout( _hide, _settings.delay );
+    } else {
+      _input.classList.remove( ERROR_CLASS );
+    }
+    _instance.dispatchEvent( 'maxCharacterChange', {
+      maxLengthExceeded: _maxLengthExceeded,
+      searchTerm: _searchTerm
+    } );
+  }
+
+  /**
+   * Event handler for input into the _input element
+   * @param {InputEvent} event The input event object
+   */
+  function _handleInput( event ) {
+    _searchTerm = event.target.value;
+    if ( _searchTerm.length >= _settings.minChars &&
+         _searchTerm.length < _settings.maxChars ) {
+      if ( _maxLengthExceeded === true ) {
+        _toggleMaxLengthError();
+      }
+      if ( _settings.url ) {
+        _throttleFetch();
+      } else {
+        _settings.filterList( _settings.list );
+      }
+    } else if ( _searchTerm.length >= _settings.maxChars ) {
+      _toggleMaxLengthError();
+    } else {
```

Comment:
Might be better organized to be:

```
} else if ( _maxLengthExceeded === true ) {
  _toggleMaxLengthError();
} else {
  _hide();
}
```

Reply:
Superseded by https://github.com/cfpb/consumerfinance.gov/pull/6537/files#r701994064

Reply:
Fixed in 608db2c (see https://github.com/cfpb/consumerfinance.gov/pull/6537#discussion_r702029297).